pkg/cbox/share/sql/sql.go

```diff
@@ -464,3 +486,69 @@ func (m *mgr) UpdateReceivedShare(ctx context.Context, ref *collaboration.ShareR
 	rs.State = f.GetState()
 	return rs, nil
 }
+
+func groupFiltersByType(filters []*collaboration.Filter) map[collaboration.Filter_Type][]*collaboration.Filter {
+	sorted := make(map[collaboration.Filter_Type][]*collaboration.Filter)
```

Comment:
I believe `sorted` is a misnomer. The end result is not sorted but rather "bundled", where all values in `filters` have been put into boxes and maps are unsorted by design. In order to avoid false assumptions in the future here I'd rename `sorted` to `b` (for bundled) 🤷

Reply:
Yeah, true. I forgot to rename it because previously the method name was `sort*`.